src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Utilities/BaseTest.cs

```diff
@@ -125,14 +125,15 @@ protected string CreateFauxAndroidSdkDirectory (string path, string buildToolsVe
 			File.WriteAllText (Path.Combine (androidSdkBuildToolsPath, IsWindows ? "aapt.exe" : "aapt"), "");
 			File.WriteAllText (Path.Combine (androidSdkToolsPath, IsWindows ? "lint.bat" : "lint"), "");
 
-			for (int i=minApiLevel; i < maxApiLevel; i++) {
-				var dir = Path.Combine (androidSdkPlatformsPath, $"android-{i}");
-				Directory.CreateDirectory(dir);
-				File.WriteAllText (Path.Combine (dir, "android.jar"), "");
+			List<ApiInfo> defaults = new List<ApiInfo> ();
+			for (int i = 10; i < 26; i++) {
+				defaults.Add (new ApiInfo () {
+					Id = i.ToString (),
+				});
```

Comment:
Only suggestion would be to only create `List<ApiInfo> defaults` when `apiLevels` is null, and then just replace the value of `apiLevels`. Right now it looks like it would create a list every time that might not be used. 

It might not really matter though, since it's just for tests.